src/components/tooltip/tooltip.js

```diff
@@ -175,8 +175,9 @@ function MdTooltipDirective($timeout, $window, $$rAF, $document, $mdUtil, $mdThe
     }
 
     function showTooltip() {
-      // Insert the element before positioning it, so we can get the position
+      // Insert the element and position at top left, so we can get the position
       // and check if we should display it
+      element.css({top: 0, left: 0});
```

Comment:
*  Why is this necessary to check computed style 'display' ?
*  Would you not want to init the position **off-screen**?
  *  eg  `element.css({top:-100, left:0})`